web/src/beta/lib/reearth-ui/components/NumberInput/index.tsx

```
@@ -47,22 +47,26 @@ export const NumberInput: FC<NumberInputProps> = ({
           } else if (max !== undefined && numericValue > max) {
             validatedValue = String(max);
-          } else {
-            validatedValue = currentValue;
           }
         }
+
         setCurrentValue(validatedValue);
+
         onChange?.(
-          currentValue === "" ? undefined : parseFloat(validatedValue)
+          currentValue === "" || isNaN(Number(currentValue))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)